tools/serverpod_cli/lib/src/analyzer/dart/endpoint_analyzers/endpoint_method_analyzer.dart

```diff
@@ -153,6 +155,34 @@ abstract class EndpointMethodAnalyzer {
 
     return null;
   }
+
+  static List<AnnotationDefinition> _parseAnnotations({
+    required Element dartElement,
+  }) {
+    return dartElement.metadata.expand<AnnotationDefinition>((annotation) {
+      var annotationName = annotation.element is ConstructorElement
+          ? (annotation.element as ConstructorElement).enclosingElement.name
+          : annotation.element?.name;
+      return switch (annotationName) {
+        'Deprecated' => [
+            AnnotationDefinition(
+              name: annotationName!,
+              arguments: [
+                annotation
+                    .computeConstantValue()
+                    ?.getField('message')
```

Comment:
Question(For my knowledge): Was there no way to fetch the parameter without knowing the field name?

Reply:
Unfortunately not, this is only an abstract superclass with very little info.